components/yaml/introduction.rst

```diff
@@ -127,20 +127,6 @@ error occurred:
 
 .. _components-yaml-dump:
 
-Objects for Mappings
```

Comment:
We should add a label to the new headline for existing deep links to the section (like you already have done it for the "Invalid Types and Object Serialization" headline).